.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,9 +18,7 @@ A clear description of what the actual behavior is.
 
 ## Steps to Reproduce the Problem
 
-1.
-2.
-3.
+1. 2. 3.
```

Comment:
Why this change? HAve you made it intentionally?

Reply:
No it automatically changed after running ```npm run format```

Reply:
Okay I just added prettierignore
https://github.com/codeforcauseorg/causefolio/blob/development/.prettierignore

Please revert the changes by copy pasting the files as it is from the main repo into places where the changes in `.github` files are made. Then rerun the `npm run format` to verify if the prettierignore is properly set up, since I didn't get time to check it personally, if you can do.


Importantly, we don't want to change github issues and PR templates due to prettier.

Reply:
Sure !